github/github.go

```diff
@@ -52,6 +52,11 @@ const (
 	// https://developer.github.com/changes/2014-12-09-new-attributes-for-stars-api/
 	mediaTypeStarringPreview = "application/vnd.github.v3.star+json"
 
+	//https://developer.github.com/v3/media/
+	mediaTypeDiff  = "application/vnd.github.v3.diff"
+	mediaTypePatch = "application/vnd.github.v3.patch"
+	mediaTypeSha   = "application/vnd.github.v3.sha"
```

Comment:
`mediaTypeSha` already exists, it's [`mediaTypeV3SHA`](https://github.com/google/go-github/blob/dccec7b5f17234b8219494c2b5b03af034e08c04/github/github.go#L44). It's probably better to add the other 2 in that same general area.

Also, the reason it's `SHA` and not `Sha` is because Go style uses consistent case for acronyms. See [here](https://github.com/golang/go/wiki/CodeReviewComments#initialisms) for rationale.